invenio_vocabularies/contrib/funders/mappings/os-v1/funders/funder-v1.0.0.json

```diff
@@ -57,6 +57,24 @@
       "country": {
         "type": "text"
       },
+      "country_name": {
+        "type": "text"
+      },
+      "location_name": {
+        "type": "text"
+      },
+      "acronym": {
+        "type": "text"
+      },
+      "status": {
+        "type": "keyword"
+      },
+      "aliases": {
+        "type": "text"
+      },
```

Comment:
comment: maybe mot for now, but we'll have to test how search behaves (probably already much better than before :) ), and possibly consider also adding a `acronym.keyword` subfield to allow for exact search matches.

Reply:
I suspect there is going to be an additional task on improving search performance. `affiliations` uses `text`, so I kept it the same here. Can I include multiple mappings?

Reply:
Agree, we should have a look at search performance across all vocabularies and tune them all at the same time. Besides adding some types for allowing for more precise queries, we should also e.g. have a look at what fields are used for free-text search, using the [`settings.index.query.default_field`](https://github.com/inveniosoftware/invenio-rdm-records/blob/7f465e49eaec29a75e14d0b38ebc2117beabfe23/invenio_rdm_records/records/mappings/os-v2/rdmrecords/records/record-v6.0.0.json#L1425-L1453)